packages/connector-jsdom/package.json

```diff
@@ -9,9 +9,9 @@
   "dependencies": {
     "@hint/utils": "^5.0.2",
     "@hint/utils-connector-tools": "^4.0.12",
+    "@types/tough-cookie": "^2.3.5",
```

Comment:
I think this should be a devDependency

Reply:
We export it via our types so we need it as a `dependency` to make it work correctly.